public/app/plugins/panel/state-timeline/utils.ts

```diff
@@ -481,7 +481,7 @@ export function getThresholdItems(fieldConfig: FieldConfig, theme: GrafanaTheme2
   for (let i = 1; i <= steps.length; i++) {
     const step = steps[i - 1];
     items.push({
-      label: i === 1 ? `< ${fmt(step.value)}` : `${fmt(step.value)}+`,
+      label: i > 1 || steps.length === 1 ? `${fmt(step.value)}+` : `< ${fmt(steps[i].value)}`,
```

Comment:
I wonder if we should use `step.value == -Infinity` rather than depend on the length 🤔

Reply:
i dont think it's possible for a base (or only) step to be anything except -Inf. but yeah, we can change the logic so it works more uniformly for a non-neginf base.